config/jobs/kubernetes/sig-gcp/gce-conformance.yaml

```diff
@@ -103,11 +82,10 @@ periodics:
       - --provider=gce
       - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]
       - --timeout=200m
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.13
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.13
 
-# manual-release-bump-required (add a job for the new version and delete the job for the oldest version)
 - interval: 6h
-  name: ci-kubernetes-gce-conformance-stable-1-10
+  name: ci-kubernetes-gce-conformance-latest-1-14
```

Comment:
seems like  
ci-kubernetes-gce-conformance-latest-1-14 and ci-kubernetes-gce-conformance-stable-1-14 job configs have merged.

Reply:
I don't think so, link to PR?

Reply:
Then this seems to be wrong. https://github.com/kubernetes/test-infra/blob/cfdefe0938f4b3ab0c962b1fe75211b1f2e741e4/config/jobs/kubernetes/sig-gcp/gce-conformance.yaml#L87

Reply:
why?

Reply:
fixed the extract line.

stable-1.14 will not work until 1.14 is released, so punt till then.

Reply:
line 88 says "latest" and line 99 says "stable".